include/alpaka/warp/WarpSingleThread.hpp

```diff
@@ -92,6 +92,22 @@ namespace alpaka
                     return predicate ? 1u : 0u;
                 }
             };
+
+            //#################################################################
+            template<>
+            struct Shfl<WarpSingleThread>
+            {
+                //-------------------------------------------------------------
```

Comment:
I was never a fan of these section comments, because I think they just add noise. But everybody else is free to convince me ;) I know we have a lot of these in alpaka.

Reply:
Me neither. @j-stephan maybe we can add this quesion to the meeting points to discuss it?

Reply:
Yes, we'll talk about it tomorrow. For now these should be kept to match the rest of alpaka.

Reply:
I'm being consistent with Ballot.

Reply:
We discussed to remove those section comments in the future, but you can keep them in this PR.